main.py

```diff
@@ -148,5 +172,85 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the Daily Leaderboard for specified date')
+async def daily(context, day):
+    # Only respond if used in a channel called 'advent-of-code'
+    if context.channel.name != 'advent-of-code':
+        return
+
+    players = get_players()
+
+    players_day = []
+
+    # Goes through all the players checking if they have data for that day and if they do adding to players_days
+    for player in players:
+        i = 0
+        #This checks they have day data
+        for item in player:
+            print(i, item)
+            i += 1
+        if (i != 5):
+            pass
+        else:
+            if (day in player[4].keys()):
+                players_day.append(player)
+            
+
+    #players_day has all people who have finished one star for that day
+    first_star = []
+    second_star = []
+
+    # Adds all the players which has 
+    for player_day in players_day:
+        if ('1' in player_day[4][day].keys()):
+            
+            first_star.append([player_day[0], int(player_day[4][day]['1']['get_star_ts'])])
```

Comment:
Should this be a tuple instead of a list? So:
`(player_day[0], int(player_day[4][day]['1']['get_star_ts']))`
instead of
`[player_day[0], int(player_day[4][day]['1']['get_star_ts'])]`

Reply:
Think I've done that last time